pyproject.toml

```diff
@@ -100,3 +100,6 @@ addopts = "--doctest-modules --ignore=setup.py --ignore=tasks.py --ignore=docs/
 
 [tool.hatch.version]
 path = "simpleflow/__init__.py"
+
+[tool.hatch.build]
+include = ["/simpleflow"]
```

Comment:
Maybe we need an `exclude` rule; if you look at the artefacts I published on test.pypi.org, there are `*.pyi~` files that were loitering around